t/myriad.t

```diff
@@ -2,18 +2,138 @@ use strict;
 use warnings;
 
 use Myriad;
+use Myriad::Commands;
 use Test::More;
 use Test::Fatal;
+use Test::MockModule;
+use Test::MockObject;
+use Future::AsyncAwait;
+use IO::Async::Test;
 
-use Scalar::Util qw(refaddr);
+sub loop_notifiers {
+    my $loop = shift;
+
+    my @current_notifiers = $loop->notifiers;
+    my %loaded_in_loop = map { ref($_)  => 1} @current_notifiers;
```

Comment:
```suggestion
    my %loaded_in_loop = map { ref => 1 } @current_notifiers;
```

Reply:
@zakame unfortunately there are two issues with that:

- it'll stringify `ref`
- it's also going to decide that you meant `map +{ 'ref' => 1 }` instead of `map BLOCK`, so would expect it to fail to compile

Reply:
@tom-binary in that case, shouldn't plain `ref()` also work?

```suggestion
    my %loaded_in_loop = map { ref()  => 1 } @current_notifiers;
```

```sh
[zakame@yoshi:~] % perl -MData::Dumper -E 'say Dumper map { ref() => 1 } ({foo=>1},[1..10])'
$VAR1 = 'HASH';
$VAR2 = 1;
$VAR3 = 'ARRAY';
$VAR4 = 1;
```

Reply:
thanks @zakame :) updated